js/ui/validation_engine.js

```diff
@@ -592,6 +592,8 @@ const ValidationEngine = {
             status: STATUS.valid,
             complete: null
         };
+        const validator = rules?.[0]?.validator;
```

Comment:
NOTE: this optional chaining need for tests passing. Rule and validator inside of it always exists in real scenario